src/firestarter/Main.cpp

```diff
@@ -215,10 +216,11 @@ Config::Config(int argc, const char **argv) {
 #ifdef FIRESTARTER_DEBUG_FEATURES
   parser.add_options("debug")
     ("allow-unavailable-payload", "")
-    ("dump-registers", "Dump the working registers on the first\nthread. Depending on the payload these are mm, xmm,\nymm or zmm. Only use it without a timeout and\n100 percent load. DELAY between dumps in secs.",
+    ("dump-registers", "Dump the working registers on the first\nthread. Depending on the payload these are mm, xmm,\nymm or zmm. Only use it without a timeout and\n100 percent load. DELAY between dumps in secs. Cannot be used with --error-detection.",
```

Comment:
why can't they be used together?

Reply:
Error detection whilst keep the high load of FS is quite complicated to achieve.
Both --error-detection and --dump-registers modify the high-load work loop of FS. Allowing both options to be used at the same time, would result in a lot more work.

Reply:
ok